### ISSUE TITLE
Update requirements.txt

### ISSUE THREAD

**Issue by alvarofpp** [UFRN-URNAI/urnai-tools#27]
Currently `requirements.txt` does not work. We need someone to create a new `requirements.txt` that works for Python 3.6.

**Comment by alvarofpp** [comment]
# Errors

## dataclasses

```
ERROR: Could not find a version that satisfies the requirement dataclasses==0.8 (from -r requirements.txt (line 10)) (from versions: 0.1, 0.2, 0.3, 0.4, 0.5, 0.6)
ERROR: No matching distribution found for dataclasses==0.8 (from -r requirements.txt (line 10))
```

## setup.py

```
Collecting wheel<1.0,>=0.23.0
  Using cached wheel-0.37.1-py2.py3-none-any.whl (35 kB)
Requirement already satisfied: setuptools>=40.3.0 in ./venv/lib/python3.8/site-packages (from google-auth==1.35.0->-r requirements.txt (line 15)) (44.0.0)
Building wheels for collected packages: clang, GPUtil, gym, mpyq, ordered-set, s2protocol, termcolor, wrapt
  Building wheel for clang (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/alvarofpp/urnai/urnai-tools/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-94x6pmjh/clang/setup.py'"'"'; __file__='"'"'/tmp/pip-install-94x6pmjh/clang/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-hdfvrbze
       cwd: /tmp/pip-install-94x6pmjh/clang/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for clang
  Running setup.py clean for clang
  Building wheel for GPUtil (setup.py) ... error
```

Same error for `GPUtil`, `gym`, `mpyq`, `ordered-set`, `s2protocol`, `termcolor` and `wrapt`.

**Comment by alvarofpp** [comment]
`ERROR: Package 'dataclasses' requires a different Python: 3.8.10 not in '>=3.6, <3.7'`, so I fixed it to `0.6`. To use `0.8`, we need to update Python to 3.8 or above.
